main-project/src/boards/entity/board.entity.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  DeleteDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+
+// fk없음, entity취합 후 생성예정
+@Entity()
+export class Board extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  no: number;
+
+  @Column({ type: 'varchar', length: 255, nullable: false })
+  title: string;
+
+  @Column({
+    type: 'tinyint',
+    width: 1,
+    default: false,
+    comment: '인원 모집 여부',
+    nullable: false,
+  })
+  done: boolean;
```

Comment:
```suggestion
  isDone: boolean;
```
네이밍 컨벤션 중요하니 꼭 기억하셔요!

Reply:
추가로 인원 모집 여부라는 코멘트를 봤는데 인원 모집 여부면 네이밍을 인원 모집을 핵심으로 작성하는 게 좋은 코드로 보여집니다!